lib/ansible/modules/cloud/amazon/ec2_lc.py

```diff
@@ -250,6 +263,8 @@ def create_launch_config(connection, module):
             if bdm.ebs is not None:
                 result['block_device_mappings'][-1]['ebs'] = dict(snapshot_id=bdm.ebs.snapshot_id, volume_size=bdm.ebs.volume_size)
 
+    if launch_configs[0].user_data != "":
+        result['user_data'] = "hidden" # Otherwise, we dump binary to the user's terminal
```

Comment:
I'm not sure we should always hide user data. I think there should either be some detection of binary vs. text, or this should be optional, since the output wouldn't be displayed unless the user was in `-vvv` or higher.